### DISCUSSION TITLE
Could someone suggest a phone for a Dexcom?

### DISCUSSION THREAD

**Discussion by OmniNegro123** [NightscoutFoundation/xDrip#2261] [General]
Hello and good day to everyone.

I am asking this because while xDrip is awesome, in the past something changed and made my current phone at the time completely unable to keep working with Dexcom. So I had to switch back to the standard receiver.

But it is about time to consider buying a new phone, so I figured I would ask here and see if someone can suggest a good phone that they know really works well with Dexcom.

Thanks in advance. Have a nice day.

**Comment by Navid200** [comment]
As Android is updated, some new issues may arise that didn't exist before.
The developers work hard to address common issues.

There are known issues experienced with some particular brands.  I would like us to document that.  I am asking the developers to help me compile such a list.  I hope I will have that soon.

You have not mentioned anything about what phone you had trouble with.  Please do.

**Reply by OmniNegro123** [reply]
Sure. I did not include my model because I was trying to not give too much information. 
I have a moto g6. (XT1925DL)
It uses Android 9 and used to work perfectly with my Dexcom. But at some point something changed. It suddenly could not maintain a connection at all, and I tried replacing the sensors and even replacing the transmitter. Nothing fixed it. So I gave up. That has been a year or two back. I am thinking it is probably a good time to consider buying a new phone and trying again. But someone out there knows a good phone that works with Dexcom and the xDrip program.

**Reply by Navid200** [reply]
I have not heard any issues with Moto G6.  G7 power?  May be.
Android 10 and above?  There are known sensitivities to wrong settings.  But, Android 9?  I am very surprised.

This is my concern:  
No matter what phone you get, you need to set it up right.  If you don't, it is very easy to have trouble.
Someone may misunderstand a list with the unrealistic idea that if a phone is on the list, it will work out of the box with no issues.  There is no phone that is like that.

These are the instructions for G6:  https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html
These are the instructions for starting a G6: https://navid200.github.io/xDrip/docs/Starting-G6.html

**Reply by Miquest** [reply]
Phones for the original Dexcom App are on this website: 
https://www.dexcom.com/de-DE/compatibility

I read that the Pixel phones are working very good with xDrip, the Google Pixel 6, for example
